hazelcast/src/test/java/com/hazelcast/internal/metrics/jmx/JmxPublisherTestHelper.java

```diff
@@ -47,7 +48,7 @@ public JmxPublisherTestHelper(String domainPrefix) throws Exception {
 
     public void assertNoMBeans() {
         Set<ObjectInstance> instances = queryOurInstances();
-        assertEquals(0, instances.size());
+        assertEquals(Arrays.toString(instances.toArray()), 0, instances.size());
```

Comment:
minor: `assertThat(instances, is(empty()))`?

Reply:
I added `Arrays.toString(instances.toArray())` to print the list of dangling MBeans in the error message. AFAIK `assertThat` without an explicit message will not print anything, will it?

Reply:
It should display the contents of the collection if not empty

Reply:
Implemented. Thanks for the suggestion.